mediaflow_proxy/extractors/fastream.py

```
@@ -0,0 +1,32 @@
+import re
```

Ruff (0.11.9)

`re` imported but unused

Remove unused import: `re`

(F401)